backend/lib/services/shoots.js

```diff
@@ -229,6 +230,18 @@ exports.remove = async function ({ user, namespace, name }) {
   return client['core.gardener.cloud'].shoots.delete(namespace, name)
 }
 
+function getDashboarUrlPath(kubernetesVersion) {
+  if (!kubernetesVersion) {
+    return undefined
+  }
+  if (semver.lt(kubernetesVersion, '1.16.0')) {
+    return '/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/'
```

Comment:
why is the old URL different from the one we had in the settings - https and the colon after kubernetes-dashboard